distributed/distributed.yaml

```diff
@@ -21,6 +21,7 @@ distributed:
     worker-ttl: null        # like '60s'. Time to live for workers.  They must heartbeat faster than this
     preload: []
     preload-argv: []
+    latency: .01            # used in calculation of transfer_time for work stealing
```

Comment:
It would be good to use times like "10 ms" here instead of float values.  (and then use `distributed.utils.parse_timedelta`)

Reply:
Similarly bandwidth should be something like `100 MiB` and we should use `distributed.utils.parse_bytes`